README.md

```diff
@@ -2,21 +2,20 @@
   <img src="etc/beerus.png" height="250" />
   <div align="center">
 
-  [![CI Action Status](https://github.com/eigerco/beerus/actions/workflows/rust-integration-tests-alchemy.yml/badge.svg)](https://github.com/eigerco/beerus/actions/workflows/rust-integration-tests-alchemy.yml)
-  [![Check Workflow Status](https://github.com/eigerco/beerus/actions/workflows/check.yml/badge.svg)](https://github.com/eigerco/beerus/actions/workflows/check.yml)
+  [![check-job-status](https://github.com/eigerco/beerus/actions/workflows/check.yml/badge.svg)](https://github.com/eigerco/beerus/actions/workflows/check.yml)
   ![starknet-version-v0.13.0](https://img.shields.io/badge/Starknet_Version-v0.13.0-2ea44f?labelColor=282d33&logo=ethereum)
   [![jsonrpc-spec-v0.6.0](https://img.shields.io/badge/JSON--RPC-v0.6.0-2ea44f?labelColor=282d33&logo=ethereum)](https://github.com/starkware-libs/starknet-specs/tree/v0.6.0)
 
   </div>
   <h1>Beerus</h1>
 
   Beerus is a Starknet Light Client inspired by and using
-  [helios](https://github.com/a16z/helios/).
+  [Helios](https://github.com/a16z/helios/).
 </div>
 
 ## News
 
-* 2024-JUN-18: "Beerus Reborn": brand new Beerus with RPC Codegen, Stateless Execution, State Proof Verification, release 0.5.0
+* 2024-JUN-18: "Beerus Reborn": brand new Beerus with RPC Codegen, Stateless Execution, State Proof Verification, release [v0.5.0](https://github.com/eigerco/beerus/releases/tag/v0.5.0)
 * 2024-FEB-29: Migrate to the [Starknet v0.6.0 OpenRPC spec](https://github.com/starkware-libs/starknet-specs/tree/v0.6.0)
 * 2025-JAN-17: [Eiger is taking over Beerus!](https://www.eiger.co/blog/eiger-taking-over-ownership-for-beerus-working-on-starknet-light-clients)
```

Comment:
I saw this after I approved the review. Everything looks good to me. Only one small thing, typo on 2025-JAN-17. Should be 2024.